plugins/services/src/js/structs/ServicePlan.ts

```diff
@@ -0,0 +1,36 @@
+import { ServicePlanResponse } from "#PLUGINS/services/src/js/data/ServicePlansClient";
```

Comment:
Can we please put the types into a types directory since the structs represent something else?